src/test/fuzz/parse_numbers.cpp

```diff
@@ -14,24 +14,9 @@ FUZZ_TARGET(parse_numbers)
 
     (void)ParseMoney(random_string);
 
-    uint8_t u8;
-    (void)ParseUInt8(random_string, &u8);
```

Comment:
It seems we now no longer have `ToIntegral` fuzz test coverage. It's a fairly simple function, so maybe that's fine, but so was `ParseIntegral`. Reckon it's worth keeping `ToIntegral` for the most common types in this harness?